package.json

```diff
@@ -8,10 +8,16 @@
     "build": "webpack --mode production",
     "devel": "webpack --mode development",
     "watch": "webpack --mode development --watch",
-    "eslint": "eslint src/",
-    "eslint:fix": "eslint --fix src/"
+    "eslint": "eslint src/ *.js",
```

Comment:
I do not get this. The Webpack ESLint plugin is checking TypeScript files too, but here is limited to plain JavaScript, why?

Reply:
This might be a bit confusing but it works as expected.

The `src/` at the beginning scans *all* files in that directory (it would scan both JS and TS files), the latter `*.js` matches only the `webpack.config.js` file in the root directory. (I was just lazy to type the full name and it is more flexible . :wink:)